lib/matplotlib/image.py

```diff
@@ -218,6 +218,19 @@ def _rgb_to_rgba(A):
 
 
 class _ImageBase(martist.Artist, cm.ScalarMappable):
+    """
```

Comment:
I had to move this up to avoid it appearing under the `AxesImage` main docstring (this doesn't affect the docs though since _ImageBase isn't in there).